.eslintrc.json

```
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
```

Mapscii doesn't use any browser APIs.
```suggestion
        "browser": false,
```